src/DependencyInjection/Compiler/AddPsyshCommandPass.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the PsyshBundle package.
+ *
+ * (c) Théo FIDRY <theo.fidry@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Fidry\PsyshBundle\DependencyInjection\Compiler;
+
+use Psy\Command\Command as PsyshCommand;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Reference;
+
+/**
+ * Compiler pass allowing to add Psysh commands dynamically
+ *
+ * @author Jérôme Vieilledent <jerome@vieilledent.fr>
+ */
+class AddPsyshCommandPass implements CompilerPassInterface
```

Comment:
I'll do it, but why do you want to enforce this? It's low level and not overridable anyway